spark-bigquery-dsv2/spark-3.5-bigquery/src/test/java/com/google/cloud/spark/bigquery/acceptance/Spark35BigNumericDataprocServerlessAcceptanceTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spark.bigquery.acceptance;
+
+import org.junit.Ignore;
+
+@Ignore // Waiting for the serverless dataproc.sparkBqConnector.uri property
```

Comment:
the property exists, however this should be ignored for now as there is no spark 3.5 runtime

Reply:
Fixed the comment